sweep/aggregator.go

```diff
@@ -86,7 +224,19 @@ func (s *SimpleAggregator) ClusterInputs(inputs pendingInputs) []inputCluster {
 	// Since the inputs that we clustered by fee rate don't commit to a
 	// specific locktime, we can try to merge a locktime cluster with a fee
 	// cluster.
-	return zipClusters(lockTimeClusters, feeClusters)
+	clusters := zipClusters(lockTimeClusters, feeClusters)
+
+	sort.Slice(clusters, func(i, j int) bool {
```

Comment:
Doesn't `zipClusters` already sort by fee rate?

Reply:
Aside from @Crypt-iQ 's point, previously we do a second sort again in `sweepPendingInputs`, so this just moves that sorting from `sweepPendingInputs` to `ClusterInputs`.